Formula/harmony-one.rb

```diff
@@ -0,0 +1,113 @@
+class HarmonyOne < Formula
+  desc "Harmony Blockchain Node"
+  homepage "https://github.com/harmony-one/harmony"
+  # CONTRIBUTORS: Update the following (url, sha256)
+  url "https://github.com/harmony-one/harmony/archive/refs/tags/v4.3.0.tar.gz"
+  sha256 "d463cd60973509f2a7e906582a09a7cd21cf3638b956f164c63e0536a2063367"
+  license "LGPL-3.0-or-later"
+
+  # CONTRIBUTORS: Versions may need to be updated according to documentation at:
+  #    -  https://github.com/harmony-one/harmony
+
+  depends_on "bash"
+  depends_on "gmp"
+  depends_on "go@1.16"
+  depends_on "jq"
+  depends_on macos: :high_sierra
+  depends_on "make"
```

Comment:
Why doesn't it work with the default make?

Reply:
because make may not be installed?

Reply:
If make is not installed Homebrew as a system will be missing dependencies it needs to run.

Reply:
> 1. The build system heavily depends on files being placed in locations that used to be in the go build system but are now just impractical. This suggests that the build system hasn't been maintained since Go switched to modules a while ago and will therefore bring issues when the next Go update comes.

Why  should the next go update matter. I originally specifically had the go versioned at 1.16 which is which version is needed to build the project. You said i should be using the latest version of go? Does every homebrew package need to be using the latest available version of all software so confused with your statements. Like you do understand that most compiled programs require specific versions of packages to build correctly right ?